optuna/visualization/_rank.py

```diff
@@ -67,9 +67,9 @@ def plot_rank(
     target: Optional[Callable[[FrozenTrial], float]] = None,
     target_name: str = "Objective Value",
 ) -> "go.Figure":
-    """Plot parameter relations as scatter plots with colors indicating ranks of objective value.
+    """Plot parameter relations as scatter plots with colors indicating ranks of target value.
```

Comment:
```suggestion
    """Plot parameter relations as scatter plots with each color indicating its rank of target value.
```
On second thoughts, I think this is more appropriate, what do you think?

Reply:
In my opinion, the sentence `colors indicating ranks of target value` is reasonable enough because the important point of this function is showing relations of target values by ranks and the former sentence is better in this point from my intuition.